Blankmon/src/application/animations/ProgressBarDecrease.java

```diff
@@ -0,0 +1,27 @@
+package application.animations;
+
+import javafx.animation.Transition;
+import javafx.beans.property.DoubleProperty;
+import javafx.util.Duration;
+
+public class ProgressBarDecrease extends Transition
+{
+	private DoubleProperty mBoundValue;
+	private double mNewValue, mOriginalValue;
+	
+	public ProgressBarDecrease(DoubleProperty boundValue, Duration duration, double newValue)
```

Comment:
Did you add the duration variable to make it take longer to drain hp on big hits?

If so I like it! 👍

Reply:
Nah, that simply tells the progress bar how long it should take to drain. In pokemon they do the same thing, that way if you take 100 damage or 2 damage, its still always the same amount of time.